Sources/App/Core/Git.swift

```diff
@@ -0,0 +1,52 @@
+import Foundation
+import ShellOut
+
+
+enum GitError: LocalizedError {
+    case invalidTimestamp
+    case invalidRevisionInfo
+}
+
+
+enum Git {
+
+    static func tag(at path: String) throws -> [Reference] {
+        let tags = try Current.shell.run(command: .init(string: "git tag"), at: path)
+        return tags.split(separator: "\n")
+            .map(String.init)
+            .compactMap(SemVer.init)
+            .map { Reference.tag($0) }
+    }
+
+    static func revList(_ reference: Reference, at path: String) throws -> CommitHash {
+        try Current.shell.run(command: .init(string: "git rev-list -n 1 \(reference)"), at: path)
+    }
+
+    static func showDate(_ commit: CommitHash, at path: String) throws -> Date {
```

Comment:
I don't think this is being used now? I only see it here and in a test. Are we keeping it around in case it's useful later? Same with `revList` above.